File: app/src/main/java/com/whatsycrrop/dpmaker/adsclass/PhotoApp.java
package com.whatsycrrop.dpmaker.adsclass;

import android.app.Activity;
import android.app.Application;
import android.content.Context;
import android.os.Bundle;
import android.util.Log;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.lifecycle.DefaultLifecycleObserver;
import androidx.lifecycle.LifecycleOwner;
import androidx.lifecycle.ProcessLifecycleOwner;
import androidx.multidex.MultiDex;

import com.google.android.gms.ads.MobileAds;
import com.google.android.gms.ads.initialization.InitializationStatus;
import com.google.android.gms.ads.initialization.OnInitializationCompleteListener;

public class PhotoApp extends Application

{


    private Activity currentActivity;
    private static final String TAG = "MyApplication";
    public static PhotoApp photoApp = null;

    @Override
    public void onCreate()
    {
        super.onCreate();
        photoApp = this;
        MobileAds.initialize(
                this,
                new OnInitializationCompleteListener()
                {
                    @Override
                    public void onInitializationComplete(
                            @NonNull InitializationStatus initializationStatus) {
                    }
                });

    }

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }




}

File: app/src/main/java/com/whatsycrrop/dpmaker/photoeditor/PhotoApp.kt
package com.whatsycrrop.dpmaker.photoeditor

import android.app.Application
import android.content.Context
import android.util.Log
import androidx.multidex.MultiDex
import com.google.android.gms.ads.MobileAds
import com.google.android.gms.ads.initialization.InitializationStatus
import com.google.android.gms.ads.initialization.OnInitializationCompleteListener

/**
 * Created by Burhanuddin Rashid on 1/23/2018.
 */
class PhotoApp : Application() {
    override fun onCreate() {
        super.onCreate()

        MobileAds.initialize(
            this
        ) {

            Log.e("TAG", "MobileAds1111112222: " )
        }
        photoApp = this




    }

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }


    companion object {
        var photoApp: PhotoApp? = null
            private set
        private val TAG = PhotoApp::class.java.simpleName
    }
}